pkg/apis/camel/v1alpha1/build_types.go

```diff
@@ -22,6 +22,7 @@ type BuildSpec struct {
 	Dependencies   []string                `json:"dependencies,omitempty"`
 	Repositories   []string                `json:"repositories,omitempty"`
 	BuildDir       string                  `json:"buildDir,omitempty"`
+	OperatorImage  string                  `json:"operatorImage,omitempty"`
```

Comment:
Maybe builder image is more appropriate ?

Reply:
Yeah, I was thinking the same